dsgrid/config/project_config.py

```diff
@@ -197,6 +203,7 @@ class DimensionMappingsModel(DSGBaseModel):
             " mappings are only supplied if a dataset's dimensions do not match the project's"
             " dimension. ",
         ),
+        updateable=False,
```

Comment:
Seems like you might update the dataset_to_project mappings. Does this preclude that?

Reply:
It's an internal field managed by dsgrid. I'm not sure if it's possible, but I may be able to make all `dsg_internal=True` fields non-updateable.